input/pagecontent/overview.md

```diff
@@ -316,17 +315,17 @@ SUSHI can be used in two modes:
 In the standalone mode, SUSHI runs independently of the IG Publisher. This is a good option to quickly check for errors when you are creating your FSH code, or if you are only interesting in creating FHIR artifacts without an IG. Creating FHIR artifacts with FSH and SUSHI in standalone mode involves the following steps:
 
 1. Populate a FSH Tank (2) with FSH files (1) containing your FSH definitions.
-2. Create a **[config.yaml](https://fshschool.org/docs/sushi/configuration/)** file (3).
-3. Run SUSHI (4). After SUSHI runs, a new directory (named **/build** by default) appears in the FSH Tank. This directory contains FHIR artifacts (5) such as profiles, extensions, value sets, and instances.
+2. Create a **[sushi-config.yaml](https://fshschool.org/docs/sushi/configuration/)** file (3).
+3. Run SUSHI (4). After SUSHI runs, a new directory (named **/fsh-generated** by default) appears in the FSH Tank. This directory contains FHIR artifacts (5) such as profiles, extensions, value sets, and instances.
```

Comment:
> After SUSHI runs, a new directory (named **/fsh-generated** by default) appears in the FSH Tank.

Again, if the tank is defined as the **fsh** folder, then this is not accurate because the tank is at **/input/fsh** and the output is at **/fsh-generated** (a sibling of **/input**).  This would be true, however, if we redefined tank to refer to the entire IG project.

Reply:
Oh...  I just expanded the view to see the context of this and I think this section needs more updating.  It refers to two modes of SUSHI: "standalone mode" and "IG mode".  I think that paradigm worked better when we supported "flat" tanks.  Now, if we want to make a distinction, it should probably be between "FSH-only mode" and "IG mode".  They both have the same project structure (w/ FSH in **input/fsh**) but the former also sets a `FSHOnly` flag in the config.

But to be honest, given that SUSHI doesn't do all the file copying and file generation it used to do (when this doc was originally written), I'm not sure it's even that valuable to separate out these two modes here.  But I'll leave that up to you.

Reply:
Should be fixed. The section is now much shorter and eliminates the two modes as well as the FSH Tank metaphor.